### PR TITLE
fix: temporary missing rill.yaml error for new projects created from rill cloud

### PR DESCRIPTION
**Pull request by k-anshul**
**Checklist:**
- [ ] Covered by tests
- [x] Ran it and it works as intended
- [x] Reviewed the diff before requesting a review
- [ ] Checked for unhandled edge cases
- [ ] Linked the issues it closes
- [ ] Checked if the docs need to be updated. If so, create a separate Linear DOCS issue
- [x] Intend to cherry-pick into the release branch
- [ ] I'm proud of this work!

### CODE REVIEW

**Review by AdityaHegde — APPROVED** [review]

File: admin/github.go [hunk 1]
```diff
@@ -209,6 +209,17 @@ func (g *githubClient) CreateManagedRepo(ctx context.Context, name string, autoI
...
+	if autoInit {
+		_, _, err = client.Repositories.CreateFile(ctx, g.managedAcct, repoName, "rill.yaml", &github.RepositoryContentFileOptions{
+			Message: github.Ptr("Add rill.yaml"),
+			Content: []byte("compiler: rillv1\n"),
+		})
+		if err != nil {
+			return nil, fmt.Errorf("failed to create rill.yaml: %w", err)
+		}
+	}
+
```
Comment (nishantmonu51) [inline admin/github.go:222]: would this affect showing of the welcome screen which we show when there is no rill.yaml file ? +cc @AdityaHegde if we need to make some more changes on UI side or treat it differently..
